Svc/CMakeLists.txt

```diff
@@ -40,6 +38,13 @@ add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/Time/")
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/TlmChan/")
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/TlmPacketizer/")
 
+# Allow and check for disabling of text loggers
+# Building these fails if FW_ENABLE_TEXT_LOGGING == 0
+if (NOT FPRIME_DISABLE_TEXT_LOGGERS)
```

Comment:
This is exactly the way I'd disable these from the build.  Two comments:

1. Add an option in `cmake/Options.cmake` this makes the switch formal, and allows you to set a default.
2. Our other switches are named in the positive. e.g. `FPRIME_ENABLE_TEXT_LOGGERS` with an appropriate default.